src/backend/tests/test-data/change-requests.test-data.ts

```diff
@@ -73,10 +73,13 @@ export const whipDeliverables: PrismaDescriptionBullet = {
 
 export const sharedChangeRequest: SharedChangeRequest = {
   crId: 1,
-  wbsNum: {
-    carNumber: 1,
-    projectNumber: 2,
-    workPackageNumber: 3
+  wbs: {
+    wbsNum: {
+      carNumber: 1,
+      projectNumber: 2,
+      workPackageNumber: 3
+    },
+    name: 'whip'
```

Comment:
i am strongly against the 'whip' testing data names becuase i have no idea what it means

Reply:
lol its supposed to be wonderwoman’s whip.

Reply:
😂